### ISSUE TITLE
Export to directory should be always available

### ISSUE THREAD

**Issue by maverick74** [tmo1/sms-ie#57]
ATM, "Export to Directory" configuration is only available if you have "enable scheduled export" ON.

But I think it would make sense to be able to define the export directory to any export/import cases (not just for "scheduled export")

**Comment by tmo1** [comment]
The reason it's set up this way is that the `Export to directory` preference does not simply set a default directory, but it enable the app to acquire permission to write its files to that directory, and this is not necessary for direct export. I guess you're suggesting that it still might make sense to use that or a similar setting to just set a default directory for direct export; I'll consider this.

**Comment by maverick74** [comment]
Yes, the idea is to be able to set it to backup for an SD Card folder, for example :)

**Comment by tmo1** [comment]
I'm not sure I understand you - is there currently a problem with setting the export directory to a location on an SD card?

**Comment by maverick74** [comment]
> I'm not sure I understand you - is there currently a problem with setting the export directory to a location on an SD card?

Oh no... i'm sorry. This is a result of testing [to many] apps. I made a confusion!
However the "default export directory" was not a confusion! Just this last part.

There is no problem in accessing SD. I can save anywhere! :)

I'm terribly sorry for my mistake :(
